### DISCUSSION TITLE
[Script request] dashboard glanceapp/glance

### DISCUSSION THREAD

**Discussion by KoekieWaus** [community-scripts/ProxmoxVE#256] [Request script]
### Application Name

Glance

### Website

https://github.com/glanceapp/glance

### Description

Is it possible to add this dashboard to the scripts? It is a nice addition and an extensive dashboard for self-hosters.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by quantumryuu** [comment]
Assign me please! Already pushed a PR 😃
